src/main/java/org/candlepin/subscriptions/ApplicationConfiguration.java

```diff
@@ -141,4 +146,19 @@ public void addResourceHandlers(ResourceHandlerRegistry registry) {
         );
     }
 
+
+    @Bean(name = "collectorRetryTemplate")
+    public RetryTemplate retryTemplate() {
+        SimpleRetryPolicy retryPolicy = new SimpleRetryPolicy();
+        retryPolicy.setMaxAttempts(4);
+
+        FixedBackOffPolicy backOffPolicy = new FixedBackOffPolicy();
+        backOffPolicy.setBackOffPeriod(2000L);
```

Comment:
Do you think it is worth making this configurable? I'm undecided.

Reply:
Meh, for now. Can revisit later.

Reply:
Ya, that's the camp I'm in as well. Figured I'd ask for other's opinions.